kuma/wiki/models.py

```diff
@@ -1327,15 +1325,11 @@ def populate_attachments(self, update_populated_field=False):
                 defaults={
                     'attached_by': revision.creator,
                     'name': revision.filename,
+                    'is_original': False,
```

Comment:
Should ``is_original`` be true if this is a new attachment? This code seems to clear that flag for all attachments.